pkg/handler/public_repositories.go

```diff
@@ -0,0 +1,44 @@
+package handler
+
+import (
+	"net/http"
+
+	"github.com/content-services/content-sources-backend/pkg/dao"
+	ce "github.com/content-services/content-sources-backend/pkg/errors"
+	"github.com/content-services/content-sources-backend/pkg/rbac"
+	"github.com/labstack/echo/v4"
+)
+
+type PublicRepositoriesHandler struct {
+	DaoRegistry dao.DaoRegistry
+}
+
+func RegisterPublicRepositoriesRoutes(engine *echo.Group, dao *dao.DaoRegistry) {
+	rph := PublicRepositoriesHandler{DaoRegistry: *dao}
+	addRoute(engine, http.MethodGet, "/public_repositories/", rph.listPublicRepositories, rbac.RbacVerbRead)
+}
+
+// ListPublicRepositories godoc
+// @Summary      List Public Repositories
+// @ID           listPublicRepositories
+// @Description  Get public repositories
```

Comment:
It may be good to describe here what a public repository is.  I'd phrase it like "A public repository is a defined repository that is available to all accounts for the purposes of searching for rpm names by url.  It does not show up via the normal repositories api."

Reply:
good idea!